mesh_aws/mesh_aws_client/tests/mesh_poll_mailbox_application_test.py

```diff
@@ -1,15 +1,19 @@
 """ Testing MeshPollMailbox application """
 
 from http import HTTPStatus
-import json
 from unittest import mock
-import requests_mock
-import boto3
+import json
+
 from moto import mock_s3, mock_ssm, mock_stepfunctions
-from mesh_aws_client import MeshPollMailboxApplication
-from mesh_aws_client.tests.mesh_testing_common import (
-    MeshTestingCommon,
+import boto3
+import requests_mock
+
+from mesh_aws.mesh_aws_client.mesh_poll_mailbox_application import (
+    MeshPollMailboxApplication,
+)
+from mesh_aws.mesh_aws_client.tests.mesh_testing_common import (
```

Comment:
Should this be mesh_aws_client?

Reply:
To recreate what you did here: https://eu-west-2.console.aws.amazon.com/lambda/home?region=eu-west-2#/functions/meshtest2-mesh-check-send-parameters?tab=code

Where `mesh_aws_client` is a directory within the deployment package, when packaging with the terraform zip resource you can't put the contents into a directory within the zip, so we need this extra directory layer which becomes the lambda function named directory when deployed to Lambda